Automated Smoketest 
Sha: 68b191f768e6a03b83e98169c963da4082188047
Build: failed
Number of Error(s): 13
collect2: error: ld returned 1 exit status
make[2]: *** [Release/bin/addScopes] Error 1
make[1]: *** [tools/addScopes/CMakeFiles/addScopes.dir/all] Error 2
collect2: error: ld returned 1 exit status
make[2]: *** [Release/bin/initldap] Error 1
make[1]: *** [tools/initldap/CMakeFiles/initldap.dir/all] Error 2
collect2: error: ld returned 1 exit status
make[2]: *** [Release/bin/ldapsecuritytest] Error 1
make[1]: *** [system/security/test/ldapsecuritytest/CMakeFiles/ldapsecuritytest.dir/all] Error 2
collect2: error: ld returned 1 exit status
make[2]: *** [Release/bin/daserver] Error 1
make[1]: *** [dali/server/CMakeFiles/daserver.dir/all] Error 2
make: *** [all] Error 2

--- a/system/security/LdapSecurity/ldapconnection.cpp
+++ b/system/security/LdapSecurity/ldapconnection.cpp
@@ -1092,6 +1092,7 @@ static __int64 getMaxPwdAge(Owned<ILdapConnectionPool> _conns, const char * _bas
     return maxPwdAge;
 }
 
+static CriticalSection  lcCrit;
 class CLdapClient : public CInterface, implements ILdapClient
 {
 private:
@@ -1118,7 +1119,6 @@ public:
         //m_defaultWorkunitScopePermission = -2;
     }
 
-    static CriticalSection  lcCrit;
     virtual void init(IPermissionProcessor* pp)
     {
         m_pp = pp;

